clients/admin-ui/src/features/data-discovery-and-detection/action-center/fields/page.tsx

```diff
@@ -262,7 +262,7 @@ const ActionCenterFields: NextPage = () => {
         </Splitter.Panel>
         {/** Note: style attr used here due to specificity of ant css. */}
         <Splitter.Panel style={{ paddingLeft: "var(--ant-padding-md)" }}>
-          <Flex vertical gap="middle" className="h-full overflow-hidden">
```

Comment:
this overflow isn't needed now that the `overflow-y-scroll` is used below, and it was causing problems with hiding the left bar on keyboard pseudo-focus state.